docs/rules/forbid-elements.md

```diff
@@ -0,0 +1,39 @@
+# Forbid certain elements (forbid-elements)
+
+You may want to forbid usage of certain elements in favor of others, (e.g. forbid all `<div />` and use `<Box />` instead). This rule allows you to configure a list of forbidden elements and to specify their desired replacements.
+
+## Rule Details
+
+This rule checks all JSX components and verifies that no forbidden elements are used. When it detects a forbidden element, the error message will contain the specified elements that you should use instead. This rule is off by default. If on, no elements are forbidden by default.
```

Comment:
If we specify `forbid: ['Modal']`, then for which of these do we warn?

```js
React.createElement('Modal');
React.createElement(Modal);
```

or something crazy like with `forbid: ['button']`:

```js
var button = 'div';
React.createElement(button);
React.createElement('button');
```

i'm guessing we should follow the same rules as JSX when translating names into either strings or identifiers